custom-routes-vis/src/main/webapp/script.js

```diff
@@ -23,15 +23,21 @@ const MarkerNames = {
   DESTINATION: 'destination',
 };
 
-// Array holding origin and destination markers.
-let originDestinationMarkers = []
-
-/** Creates the world map with markers. */
-function createWorldMapWithMarkers() {
-  map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
-  
+let originDestinationMarkers = [];
+let displayedRoutes = [];
+const routeColors = ['#ff0000', '#eb8f1e', '#3299d1'];
+let selectedRouteNum = 0;
+var map;
+
+/** Creates the world map and markers. */
+function initMapWithMarkers() {
+  let chicago = new google.maps.LatLng(41.850033, -87.6500523);
+  let mapOptions = {
+    zoom: 7,
+    center: chicago,
```

Comment:
Why Chicago?

Reply:
Just random, but I think we should make it somewhere instead of in the middle of nowhere